libs/dh/metering-point/feature-overview/src/lib/breadcrumb/dh-breadcrumb.component.html

```diff
@@ -0,0 +1,23 @@
+<!--
+@license
+Copyright 2021 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<nav class="breadcrumb">
+  <a [routerLink]="meteringPointAbsolutePath">{{
+    'meteringPoint.overview.breadcrumb.meteringPoints' | transloco
```

Comment:
question: I see now that there's no `read` parameter for the Transloco pipe. Having the scope in a separate parameter using the Transloco structural directive seemed more clean in my opinion. What's your take on this?

Reply:
We should probably figure out how to use translation texts in tests before deciding on a preferred Transloco declarable. See other thread.

Reply:
> Having the scope in a separate parameter using the Transloco structural directive seemed more clean in my opinion. 

I agree. It was using the directive initially but I was wrapping the whole template. Maybe we can still use the directive but only wrap the link instead, so the other data is shown. A downside is that there's a jump if the translation takes some time to load and the is shown after the GSRN number is already there.

Reply:
Documentation: https://ngneat.github.io/transloco/docs/unit-testing/

Reply:
@dzhavat @LayZeeDK lets have a talk about this after daily tomorrow, I've already done a temp translation testing module in my branch